testsuite/manualtests/test_conngen_interface.sli

```diff
@@ -0,0 +1,111 @@
+/*
```

Comment:
Why is this test in manualtests?

Reply:
With the upcoming merge of #774, `manualtests` will be gone. I can change this file to an example instead until we have a) and b) fulfilled. What do you think?

Reply:
@jougs Turning it into an example for now would be fine for me, provided we create a follow-up ticket reminding us to create the test (with Milestone 3.0). I suppose this also depends on #817 being fixed?